datacats/docker.py

```diff
@@ -24,7 +23,7 @@
 from docker import Client
 from docker.constants import DEFAULT_DOCKER_API_VERSION
 from docker.utils import kwargs_from_env, compare_version, create_host_config, LogConfig
-from docker.errors import APIError
+from docker.errors import APIError, TLSParameterError
```

Comment:
new imports from docker mean we might need to update the required version of docker-py, please check that